pontos/nvd/api.py

```diff
@@ -55,17 +55,28 @@ def now() -> datetime:
     return datetime.now(tz=timezone.utc)
 
 
-def format_date(date: datetime) -> str:
+def format_date(
+    date: datetime,
+    fallback_timezone: timezone = timezone.utc,
+    timespec: str = "milliseconds",
```

Comment:
Optional arguments should always be passed as keywords.
```suggestion
    *,
    fallback_timezone: timezone = timezone.utc,
    timespec: str = "milliseconds",
```